k8s/charts/seaweedfs/values.yaml

```diff
@@ -381,6 +387,15 @@ volume:
   sidecars: []
   initContainers: ""
 
+  # Example for use when using more than 1 volume server replica
+  # extraVolumeMounts: |
+  #   - name: drive
+  #     mountPath: /drive
+  #     subPathExpr: $(POD_NAME)
+  # extraVolumes: |
+  #   - name: drive
+  #     hostPath:
+  #       path: /var/mnt/
   extraVolumes: ""
```

Comment:
`extraVolumes` and `extraVolumeMounts` should not be strings?

Reply:
That was my first thought too. But when I tried to deploy I got this error: 
```
Error: template: infra/charts/seaweedfs/templates/volume-statefulset.yaml:213:26: executing "infra/charts/seaweedfs/templates/volume-statefulset.yaml" at <.Values.volume.extraVolumeMounts>: wrong type for value; expected string; got []interface {} Use --debug flag to render out invalid YAML
```

So then I changed it to a string and it worked. I wasn't super surprised as it's a common enough pattern in k8s, especially when extra variable replacement needs to be done (like the $(POD_NAME) here). 

Happy to pull more info if you're curious, that just was the relevant bit.